Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -499,6 +499,30 @@ mode would look as follows:
 
 .. _XDP acceleration:
 
+Socket LoadBalancer Bypass in Pod Namespace
+*******************************************
+
+Cilium has built-in support for bypassing the socket-level LoadBalancer and falling back
+to the tc LoadBalancer when a custom redirection/operation relies on the original ClusterIP
+within pod namespace (e.g., Istio side-car).
+
+Setting ``loadBalancer.hostNamespaceOnly`` allows to enable this bypassing mode when set
+to ``true``. When enabled, this circumvents socket rewrite in the ``connect()`` syscall bpf
```

Comment:
> When enabled, this circumvents socket rewrite in the ``connect()``

I think this is not complete - `__sock{4,6}_xlate_fwd()` is invoked by the `sock{4,6}_sendmsg()` too. So, it applies to the `sendmsg()` syscall too which can be used for non-connected UDP.

Reply:
ah makes sense, added `sendmsg()`.